cogment_verse/processes/model_registry.py

```diff
@@ -19,7 +19,7 @@
 
 
 def create_model_registry_service(work_dir, model_registry_cfg, services_directory):
-    model_registry_data_dir = os.path.join(work_dir, "model_registry")
+    model_registry_data_dir = os.path.join(work_dir, ServiceType.MODEL_REGISTRY.value)
```

Comment:
It's a little weird to use the a service type value as a path here, maybe a separate constant should be used if you wanna extract those.

Reply:
Let's go for separate constant.